src/data/faq_de.json

```
@@ -1428,12 +1428,12 @@
                         ]
                     },
                     {
-                        "title": "Wie wird der Stand in GitHub auf die App in den App Stores übertragen?",
+                        "title": "Wie wird der Entwicklungsstand in GitHub auf die App in den App Stores übertragen?",
                         "anchor": "dev_status",
                         "active": true,
                         "textblock": [
-                            "Die App wird über Standardwerkzeuge (z.&nbsp;B. Xcode, Android Studio) erstellt und dann in die jeweiligen App Stores hochgeladen.",
-                            "Das Thema, ob der Code in GitHub wirklich mit der App in den jeweiligen App Stores übereinstimmt ('Reproducible Builds'), haben wir auf dem Schirm und werden entsprechende Informationen veröffentlichen, sobald sie verfügbar sind. Details zum Fortschritt findest Du im entsprechenden <a href='https://github.com/corona-warn-app/cwa-documentation/issues/14' target='_blank' rel='noopener noreferrer'>GitHub-Issue</a>."
+                            "Die App wird mit Standardwerkzeuge (z.B. Android Studio für Android, Xcode für iOs) entwickelt. Releases werden auf GitHub öffentlich angekündigt (<a href='https://github.com/corona-warn-app/cwa-app-android/releases' target='_blank' rel='noopener noreferrer'>Android</a>, <a href='https://github.com/corona-warn-app/cwa-app-ios/releases' target='_blank' rel='noopener noreferrer'>iOS</a>). Der Quelltext eines Releases wird zu der App-Anwendung kompiliert, welche in den jeweiligen App Stores von Google und Apple veröffentlicht wird.",

```

Plural, so "Standardwerkzeugen", "s" of "iOS" spells with a capital letter.

```suggestion
                            "Die App wird mit Standardwerkzeugen (z.B. Android Studio für Android, Xcode für iOS) entwickelt. Releases werden auf GitHub öffentlich angekündigt (<a href='https://github.com/corona-warn-app/cwa-app-android/releases' target='_blank' rel='noopener noreferrer'>Android</a>, <a href='https://github.com/corona-warn-app/cwa-app-ios/releases' target='_blank' rel='noopener noreferrer'>iOS</a>). Der Quelltext eines Releases wird zu der App-Anwendung kompiliert, welche in den jeweiligen App Stores von Google und Apple veröffentlicht wird.",
```